lib/modules/manager/api.ts

```diff
@@ -40,6 +40,7 @@ import * as hermit from './hermit';
 import * as homebrew from './homebrew';
 import * as html from './html';
 import * as jenkins from './jenkins';
+import * as jsonata from './json-jsonata';
```

Comment:
Why the choice of `json-jsonata` name instead of `jsonata`? e.g. were you anticipating needing something like `yaml-jsonata` later?

Reply:
Because of that and because I wanted to minimize the confusion since there is already a module named `jsonata`, which is the one provided by the jsonata npm package. 

We can change it, but this is not going to be exposed externally, only what we have in documentation and configuration will initially matter, won't it?

What do you think about renaming later when we have unified the managers in one? I think names will become more obvious.

Or what would you suggest?

Reply:
renaming later needs major version bump and causes a lot migration code, so please do renaming now